pkg/skaffold/initializer/testdata/init/helm-deployment/skaffold.yaml

```diff
@@ -8,5 +8,3 @@ deploy:
     releases:
```

Comment:
I'm not sure but I think this should maybe have apiVersion:skaffold/v3alpha2 (currently skaffold/v2beta5)